1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature

```diff
@@ -88,4 +88,56 @@ Feature: Owner navigates to the Landing Page for their Resource
         When the Owner uses the option to transfer Ownership of current Resource
         Then the Owner must acknowledges that this is a final action
         And the Owner must select a new Owner
-        And the ownership is transfered to new Owner
\ No newline at end of file
+        And the ownership is transfered to new Owner
+
+    @TEST_NP-4200
+    @1235
+    @test
+    Scenario: Owner navigates to the Landing Page and publish a Registration with a drafted DOI
+        Given that the Owner navigates to a Registration's Landing Page
+        And the Registration has status Draft
+        And the Registration has a Draft DOI
+        When the Owner publishes the Registration
+        Then the Landing Page for Registration is displayed
+        And the option to draft a DOI is displayed as "DOI pending" and is disabled
+        And the Landing Page for Registration lists the Draft DOI
+        And the Draft DOI is still not a link
```

Comment:
det vil sikkert være en link, men den vil ikke peke til noe

Reply:
Poenget er at reserved DOI er synlig på landingpage - for registrator. Men, den vil ikkje peike på noko (vi gi ein 404). Derfor kan den evt. kun stå som tekst. I dag presentere vi den som en lenke, som ikkje fungerer - den ender med ein 404.